glossary.md

```diff
@@ -203,6 +203,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 ## I
 | English                    | Tiếng Việt                   | Thảo luận tại                                |
 |----------------------------|------------------------------|----------------------------------------------|
+| ill-conditioned problem    | bài toán có điều kiện xấu    |                                              |
```

Comment:
từ `ill-conditioned` mình thấy đã có trong từ điển là `điều kiện xấu` hay thấy trong ngữ cảnh toán rồi mà nhỉ.
Mình cũng có thể cho thêm vào `glossary` nếu sợ nó chưa phổ biến và dễ bị nhầm, nhưng không cần phải giới hạn thêm chữ `problem`.
```suggestion
| ill-conditioned    |  (có) điều kiện xấu    |                                              |
```